src/popup/popup.js

```diff
@@ -239,8 +238,6 @@ elements.filterApplyButton.addEventListener('click', () => {
 elements.searchIcon.addEventListener('click', () => {
   inputText = elements.inputField.value.trim().replace('/[ ]+/g', ' ');
   pageNo = 1;
-
-  checkInputError(inputText, 'error-message');
```

Comment:
@cristomathew You should not remove the `checkInputError` function. It is an important one. You just need to remove the 'error-message' argument because if you see in the definition of the function, the second argument is no longer needed.

Reply:
K sorry when u mentioned I forgot to remove it i thought u meant the whole thing. 😅